docs/ref/django-admin.txt

```diff
@@ -601,12 +601,13 @@ The :djadmin:`dumpdata` command can be used to generate input for ``loaddata``.
 Compressed fixtures
 ~~~~~~~~~~~~~~~~~~~
 
-Fixtures may be compressed in ``zip``, ``gz``, or ``bz2`` format. For example::
+Fixtures may be compressed in ``zip``, ``gz``, ``bz2`` or ``xz`` format.
+For example::
 
     django-admin loaddata mydata.json
 
-would look for any of ``mydata.json``, ``mydata.json.zip``,
-``mydata.json.gz``, or ``mydata.json.bz2``. The first file contained within a
+would look for any of ``mydata.json``, ``mydata.json.zip``, ``mydata.json.gz``,
+``mydata.json.bz2`` or ``mydata.json.xz``. The first file contained within a
```

Comment:
Serial comma should stay before 'or' (see https://en.wikipedia.org/wiki/Serial_comma).

Reply:
`versionchanged`  annotation is missing, e.g.:
```
.. versionchanged:: 3.2

    Support for XZ archives (``.xz``) and LZMA archives (``.lzma``) was added.
```